### DISCUSSION TITLE
NEW LOGIN METHOD FOR TWITTER ACCOUNTS SUSPENDED OR CLOSED

### DISCUSSION THREAD

**Discussion by Akrepkapo** [IBAX-io/go-ibax#2081] [Ideas]
Problem : Accounts whose Twitter account has been permanently disabled or suspended cannot log into the ibax network account.

Solution : Dear ibax network team, as you know, we have linked our githup account to ibax network for airdrop. We can integrate the githup login method for accounts whose Twitter account has been closed. And after logging in via githup, we can update our Twitter account with the new account.

According to my research results, the number of victims whose Twitter accounts were closed: 25-50.

Thanks in advance.

@BadAssKyle 
@yolandadadada 
@akibax 
@ibaxio 
@group-monkey 
@ysfcndgr 
@scottafk

**Comment by MrDino3813** [comment]
Login method on what platform GitHub or IBAX Account

**Reply by Akrepkapo** [reply]
we previously linked our githup accounts to ibax for airdrop. This is how those who have their Twitter account closed can access their accounts

**Reply by Akrepkapo** [reply]
???

**Reply by Akrepkapo** [reply]
Sorun tam olarak neydi?

**Reply by Gazarta** [reply]
Hocam benim Twitter hesabı askıya alındı yardımcı olur musunuz
KGazarta hesab adı

**Comment by Brews120** [comment]
How to login sir,my Twitter account has been suspended

**Reply by Akrepkapo** [reply]
githup login method will be integrated for this. @yolandadadada  is interested in this issue.

**Comment by Brews120** [comment]
My Twitter account  @Fendi_120120

**Comment by Heruamat** [comment]
![Screenshot_2022-12-16-02-16-34-64](https://user-images.githubusercontent.com/88230083/207949316-85e49794-f59b-4373-908f-9bbab1c5f9fb.jpg)

My twitter @Raph78209838

Hallo admin , there was an error on my IBAX account regarding Twitter, previously I had followed the IBAX Twitter account and I never unfollowed it, but when I re-logined, I suddenly my IBAX points decreased by 2000 points with a record of unfollowing, even though I didn't do it at all, is this a bug?

**Reply by Akrepkapo** [reply]
Please talk to @yolandadadada  about the issue.

**Comment by Gkhndkngl** [comment]
Merhabalar Twitter hesabım askıya alınmış giriş yapamıyorum bu konuda yardımcı olursanız çok sevinirim şimdiden teşekkürler 
![Screenshot_2022-12-25-01-26-42-218_com android chrome](https://user-images.githubusercontent.com/121364953/209453633-df962599-667b-4b89-9d02-d4724b00d316.jpg)

**Comment by msport444** [comment]
Hello, my twitter account was suspended. Please help me.

**Comment by zeland2323** [comment]
Thanks
